client/http/interface.go

```diff
@@ -1024,3 +1026,29 @@ func (c *client) GetKeyspaceMetaByName(ctx context.Context, keyspaceName string)
 	}
 	return &keyspaceMetaPB, nil
 }
+
+func (c *client) GetGCSafePoint(ctx context.Context) (ListServiceGCSafepoint, error) {
```

Comment:
Each exported function should contain a line of comments.

```suggestion
// GetGCSafePoint ...
func (c *client) GetGCSafePoint(ctx context.Context) (ListServiceGCSafepoint, error) {
```

Reply:
will do!